src/client/lib/utils.js

```diff
@@ -101,7 +101,7 @@ export var async = (gen) => {
  * @param  {String} text  Optional text content
  * @return {HTMLElement}
  */
-export var createElement = function(type, attrs={}, text=null) {
+export let createElement = function(type, attrs={}, text=null) {
```

Comment:
'export' is only available in ES6 (use esnext option).<br>'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'default parameters' is only available in ES6 (use esnext option).